backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/report/AD_Process_StepDef.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * #%L
+ * de.metas.cucumber
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.cucumber.stepdefs.report;
+
+import de.metas.cucumber.stepdefs.DataTableRow;
+import de.metas.cucumber.stepdefs.DataTableRows;
+import de.metas.cucumber.stepdefs.util.IdentifiersResolver;
+import de.metas.i18n.Language;
+import de.metas.process.AdProcessId;
+import de.metas.process.IADPInstanceDAO;
+import de.metas.process.IADProcessDAO;
+import de.metas.process.ProcessCalledFrom;
+import de.metas.process.ProcessInfo;
+import de.metas.report.server.LocalReportServer;
+import de.metas.report.server.OutputType;
+import de.metas.report.server.ReportResult;
+import de.metas.util.Services;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.archive.api.ArchiveRequest;
+import org.adempiere.archive.api.IArchiveBL;
+import org.adempiere.util.lang.impl.TableRecordReference;
+import org.compiere.model.I_AD_Process;
+import org.compiere.util.Env;
+import org.springframework.core.io.ByteArrayResource;
+
+import static de.metas.cucumber.stepdefs.util.IdentifiersResolver.RECORD_ID;
```

Comment:
What's the point of defining that constant RECORD_ID in IdentifiersResolver, when it's never used in IdentifiersResolver but only here?

Reply:
I thought in context of Identifier resolver we could use it in multiple places, to avoid defining it in multiple places later on